webapp/src/linterChecks/Signe.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SpinButton from '../components/atoms/SpinButton/SpinButton';
+import { Link, Outlet } from 'react-router-dom';
```

Comment:
TIL about Outlet! I haven't used it manually before!

Reply:
I was _yesterday_ years old when I found out about Outlet! You'll have to let me know you've used it in the past. 😀